main_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"strings"
+	"testing"
+
+	"github.com/mendersoftware/go-lib-micro/log"
+)
+
+var runAcceptanceTests bool
+
+// used for parsing '-cli-args' for urfave/cli when running acceptance tests
+// this is because of a conflict between urfave/cli and regular go flags required for testing (can't mix the two)
+var cliArgsRaw string
+
+func init() {
+	// disable logging while running tests
+	log.Log.Out = ioutil.Discard
```

Comment:
what about making this configurable by using this:
https://github.com/mendersoftware/go-lib-micro/blob/master/log/testing/log.go#L26